docs/src/examples/txnkv.md

```diff
@@ -0,0 +1,69 @@
+# TxnKV
```

Comment:
@zz-jason This is a little bit too low-level... Should we give example for txnkv?

Reply:
It's OK, we can update this document once there are easy ways to interact with transactions.